changelog.d/10968.bugfix

```diff
@@ -0,0 +1 @@
+Fix `/admin/whois/{user_id}` endpoint, which was broken in v1.44.0rc1.
```

Comment:
Are these changelogs meant to be in here?

(Guessing they're already merged to develop?)

Reply:
They're from PRs #10968 and #10972. One's in the release branch and the other's not merged yet. Without those PRs the types don't check out.

Reply:
Ahhh, I should have seen your description and only looked at the last commit.